util/which-key/README.txt

```diff
@@ -0,0 +1,9 @@
+This module enables functionality similar to `which-key-mode` in
+emacs.  When a keymap is enabled by pressing a key, a list of possible
+keysequences are shown to the user, similar to the help menu (`C-t ?`)
+For example, if I press `C-t` the commands in the root-map are shown
```

Comment:
The first person should not used in documentation. Plus you are already using third person in the previous sentence. I prefer you used second person but either second or third person are fine

ej.
```
For example, when we press C-t
For example, when C-t is pressed
```